package-lock.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
Do we need to check in the `node_modules` directory? Ideally it is bundled into the main runtime artifacts (index.js). 

I am looking at the azure/login repo.

I see a couple of things:

1. The main branch is different from the release branch. The main branch does not have build output (such as `dist` -- they seem to call it `lib`)
2. No node_modules is checked in

I think we should consider following this approach so that our main branch does not become cluttered with build output.